src/plugins/paginate/paginate.js

```diff
@@ -0,0 +1,262 @@
+/**
+ * @title WET-BOEW Tag filter
+ * @overview Filter based content tagging
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @duboisp
+ */
+( function( $, window, document, wb ) {
+"use strict";
+
+let i18n, i18nText;
+
+const componentName = "wb-paginate",
+	selector = ".provisional." + componentName,
+	initEvent = "wb-init" + selector,
+	$document = wb.doc,
+	filterClass = "wb-fltr-out",
+	pgFilterOutClass = "wb-pgfltr-out",
+	tagFilterClass = "wb-tgfltr-out",
+	pagerClass = "wb-paginate-pager",
+	pageData = "data-pagination-idx",
+	notFilterClassSel = ":not(." + filterClass + "):not(." + tagFilterClass + ")",
+
+	init = function( event ) {
+		const elm = wb.init( event, componentName, selector );
+
+		if ( elm ) {
+			var userConf = JSON.parse( elm.getAttribute( "data-wb-paginate" ) ),
+				paginationElm,
+				elmTagName = elm.nodeName,
+				setDefault,
+				appendUI,
+				defaults = {
+					lst: {
+						selector: "li"
+					},
+					grp: {
+						selector: "> *",
+						appendUI: true
+					},
+					tbl: {
+						selector: "tr",
+						section: ":scope > tbody"
+					}
+				};
+
+			if ( !i18nText ) {
+				i18n = wb.i18n;
+				i18nText = {
+					prv: i18n( "prv" ),
+					nxt: i18n( "nxt" )
+				};
+			}
+
+			// Setup configurations
+			if ( [ "UL" ].indexOf( elmTagName ) >= 0 ) {
```

Comment:
If that list `[ "UL" ]` limited to one element or the intend is to add a few more?